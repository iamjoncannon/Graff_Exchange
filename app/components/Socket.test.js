
// https://medium.com/@tozwierz/testing-socket-io-with-jest-on-backend-node-js-f71f7ec7010f

describe("Socket", ()=>{

    it('1) renders successfully with data from redux store', ()=>{

        // expect(true).toBeTruthy()

        // let tree = component.toJSON();
        
        // expect(tree).toMatchSnapshot();
    })

})