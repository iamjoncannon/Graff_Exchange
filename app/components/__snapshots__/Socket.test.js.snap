// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Socket 1) renders successfully with data from redux store 1`] = `
<div
  style={
    Object {
      "height": "0",
      "width": "0",
    }
  }
/>
`;
